### PR TITLE
Init WoTMotion german caption

### PR DESCRIPTION
**Pull request by sebastiankb**

### CODE REVIEW

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 1]
```diff
@@ -0,0 +1,251 @@
+WEBVTT
+
+1
+00:00:00.500 --> 00:00:03.000
+Ein Obstbauer sucht nach einer Möglichkeit
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:5]: sucht nach Möglichkeiten

both ok.

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+4
+00:00:08.500 --> 00:00:13.000
+plant der Landwirt Feuchtigkeits- und Temperatursensoren sowie 
+ein Sprinklersystem zu installieren.
+
+5
+00:00:13.500 --> 00:00:16.000
+Wettervorhersagedaten 
+von einem Online-Webdienst
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:26]: Online-Wetterservice ?

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+Bewässerungsbedarf vorherzusagen.
+
+7
+00:00:20.500 --> 00:00:25.000
+Der Landwirt beschließt, einen 
+IT-Berater zu engagieren.
+
+8
+00:00:25.500 --> 00:00:29.000
+Er möchte die Entwickler seines bisherigen Internetauftritts nutzen
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:40]: Entwickler nutzen? maybe better 

Er möchte die Entwickler seines bisherigen Internetauftritts beauftragen

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+
+9
+00:00:29.500 --> 00:00:33.000
+Die Entwickler der Webagentur 
+freuen sich auf die neue Herausforderung
+
+10
+00:00:29.500 --> 00:00:36.000
+und sind bereit, ihr IoT-Wissen 
+ihr IoT-Wissen zu erweitern.
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:51]: "IoT-Wissen" twice

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+
+11
+00:00:36.500 --> 00:00:41.000
+Sie sind Experten sowohl für Web-Frontend- als auch
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:55]: "Web-Frontend-" --> "Web-Frontend" ?

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+
+13
+00:00:45.500 --> 00:00:49.000
+die bei einem früheren IoT-Projekt 
+aufgetreten sind. 
+
+14
+00:00:49.500 --> 00:00:53.000
+Nach einer Online-Recherche 
+für eine gute Lösung,
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:71]: "für eine gute Lösung," really needed? Sounds better without I think.

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+
+15
+00:00:53.500 --> 00:00:58.000
+finden die Entwickler Informationen 
+über das Web of Things, WoT.
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:76]: Shall we mention once the German term? a la

über das Web of Things (dt. Web der Dinge), WoT.

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 8]
```diff
@@ -0,0 +1,251 @@
...
+im Bereich der Landwirtschaft.
+
+21
+00:01:19.500 --> 00:01:23.000
+Die Entwickler sind glücklich 
+viele Informationen zu finden
+
+22
+00:01:23.500 --> 00:01:27.000
+von WoT-Tutorials wie auf thingweb.io,
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:110]: "wie z.B. WoT-Tutorials auf thingweb.io," ?

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 9]
```diff
@@ -0,0 +1,251 @@
...
+
+24
+00:01:29.500 --> 00:01:34.000
+wie HTTP, MQTT, CoAP 
+und Modbus,
+
+25
+00:01:34.500 --> 00:01:39.000
+sowie einem Browser-Bundle, können die Entwickler leicht eine 
+eine Full-Stack-IoT-Lösung entwickeln.
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:126]: "wie HTTP, MQTT, CoAP und Modbus realisiert werden."

"Ein Browser Bundle erleichtert die Entwicklung einer Full-Stack-IoT-Lösung."

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 10]
```diff
@@ -0,0 +1,251 @@
...
+
+26
+00:01:39.500 --> 00:01:42.000
+Ihre Lösung verbindet 
+alle Geräte
+
+27
+00:01:42.500 --> 00:01:45.000
+und beinhaltet eine großartige Benutzeroberfläche
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:135]: "großartig" --> sounds to good ;-)
maybe "intuitive Benutzeroberfläche"

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 11]
```diff
@@ -0,0 +1,251 @@
...
+
+26
+00:01:39.500 --> 00:01:42.000
+Ihre Lösung verbindet 
+alle Geräte
+
+27
+00:01:42.500 --> 00:01:45.000
+und beinhaltet eine großartige Benutzeroberfläche 
+die mit Web-Technologie aufgebaut ist.
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:136]: TechnologieN ... add n ?

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 12]
```diff
@@ -0,0 +1,251 @@
...
+
+28
+00:01:45.500 --> 00:01:49.000
+Anstatt eine einmalige 
+proprietäre Lösung zu schaffen,
+
+29
+00:01:49.500 --> 00:01:54.000
+verwenden die Entwickler die WoT-Methodik 
+um IoT-Builiding Blocks zu erstellen.
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:146]: typo "Building"

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 13]
```diff
@@ -0,0 +1,251 @@
...
+
+35
+00:02:18.500 --> 00:02:21.000
+ist die Thing Description, 
+auch bekannt als TD.
+
+36
+00:02:21.500 --> 00:02:26.000
+Die TD ist vergleichbar 
+mit einer index.html für Things.
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:181]: index.html für Dinge?

In general. Shouldn't the English text also read as
"...comparable to an index.html for Websites!??"

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 14]
```diff
@@ -0,0 +1,251 @@
...
+
+36
+00:02:21.500 --> 00:02:26.000
+Die TD ist vergleichbar 
+mit einer index.html für Things. 
+
+37
+00:02:21.500 --> 00:02:30.000
+Verschiedene Fragen können beantwortet werden
+ durch die Informationen in der TD,
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:186]: leading space... not sure if this causes issues?

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 15]
```diff
@@ -0,0 +1,251 @@
...
+
+39
+00:02:40.500 --> 00:02:44.000
+Klassische IoT-Projekte 
+verursachen in der Regel hohe Kosten,
+
+40
+00:02:44.500 --> 00:02:47.000
+insbesondere aufgrund 
+durch den hohen Integrationsaufwand
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:201]: des hohen Integrationsaufwandes  ?

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 16]
```diff
@@ -0,0 +1,251 @@
...
+
+41
+00:02:47.500 --> 00:02:51.000
+der unterschiedlichen Datenmodelle 
+und Protokollen der Geräte bzw. Dienste.
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:206]: Protokolle ? .. without n

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 17]
```diff
@@ -0,0 +1,251 @@
...
+
+42
+00:02:51.500 --> 00:02:57.000
+Im Gegensatz dazu erstellen die Entwickler mit WoT 
+die IoT-Lösung in nur wenigen Sprints,
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:211]: I don't really like "in nur wenigen Sprints" but I do not really have a good alternative.
Maybe somethink like

"in nur einigen Schritten"

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 18]
```diff
@@ -0,0 +1,251 @@
...
+
+43
+00:02:57.500 --> 00:03:03.000
+indem sie WoT TDs, Protokollbindungen 
+und Scripting mit ihrer Web-Expertise kombinieren. 
+
+44
+00:03:03.500 --> 00:03:07.000
+Dies führt zu einer Reduzierung der Entwicklungs
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:220]: add traling "-" ? --> "Entwicklungs-"

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 19]
```diff
@@ -0,0 +1,251 @@
...
+in IoT
+
+47
+00:03:11.500 --> 00:03:17.000
+und betrachten sich von nun an als Experten 
+sowohl in Web- als auch in IoT-Technologien.
+
+48
+00:03:17.500 --> 00:03:22.000
+Auf Basis von WoT konnte das IoT-Projekt des Landwirts
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:240]: I think "Landwirts" is fine.. but we used "Landwirtes" before and so i think we should use it here also.

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt [hunk 20]
```diff
@@ -0,0 +1,251 @@
...
+
+49
+00:03:22.500 --> 00:03:27.000
+Jetzt sind die Entwickler bereit für die 
+nächste IoT-Herausforderung.
+
+50
+00:03:27.500 --> 00:03:41.000
+Und ja, WoT ist DER Ansatz 
+die sie verwenden werden!
```
Comment (danielpeintner) [inline videos/WoT-Explainer-Video/WOTMotion.de.vtt:251]: typo?
den sie ...

**Review by danielpeintner — COMMENTED** [review]
I have a lot of minor comments. Most of them are just a matter of taste... hence feel free to ignore them.

There are some typos that need to be fixed though.

**Comment by sebastiankb** [comment]
@danielpeintner many thanks I just updated the text.

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt (hunk 6 above)
Comment (danielpeintner) [reply videos/WoT-Explainer-Video/WOTMotion.de.vtt:71]: Still there?

**Comment by danielpeintner** [comment]
> @danielpeintner many thanks I just updated the text.

@sebastiankb I think GitHub _hided_ some of my comments... please have a look

File: videos/WoT-Explainer-Video/WOTMotion.de.vtt (hunk 15 above)
Comment (danielpeintner) [reply videos/WoT-Explainer-Video/WOTMotion.de.vtt]: "insbesondere aufgrund (durch) des hohen Integrationsaufwandes"

I would rather say it without "durch" but I don't really mind...

**Review by danielpeintner — APPROVED** [review]
LGTM

**Comment by sebastiankb** [comment]
from today's marketing call we merge this PR

**Pull request merged**